app/models/concerns/user_concern.rb

```diff
@@ -0,0 +1,9 @@
+module UserConcern
```

Comment:
Study difference between concern and decorators. Also create a decorator for study purpose. Rename this concern to generic. This should be something like `TokenableConcern`